### ISSUE TITLE
Fix code in Google Cloud Storage Integration Wiki Page

### ISSUE THREAD

**Issue by leoncvlt** [VeliovGroup/Meteor-Files#571]
I have copied the example code you have in the Google Cloud Storage Integration Wiki page, and it worked flawlessly when uploading single image. However, I had problems when using it with files with subversions (such as when generating thumbnails with the example in the Image Processing page) - specifically it was uploading the original file twice.

I think the issues lies in here:
```js
onAfterUpload: function(fileRef) {
    // In the onAfterUpload callback, we will move the file to Google Cloud Storage
    var self = this;
    _.each(fileRef.versions, function(vRef, version){
      // We use Random.id() instead of real file's _id
      // to secure files from reverse engineering
      // As after viewing this code it will be easy
      // to get access to unlisted and protected files
      var filePath = "files/" + (Random.id()) + "-" + version + "." + fileRef.extension;
      // Here we set the neccesary options to upload the file, for more options, see
      // https://googlecloudplatform.github.io/gcloud-node/#/docs/v0.36.0/storage/bucket?method=upload
      var options = {
        destination: filePath,
        resumable: true
      };

      bucket.upload(fileRef.path, options, function(error, file){
...
```

it's using `_.each` to go through each version of the file, but when specifying the source file to upload it uses 'fileRef.path' which **is the same for each version**. To upload the correct file, it should be using `vRef.path`, which is the specific path for that version.

This is further confirmed by looking at the object structure of the fileRef object passed to the function:

```js
{ size: 829970,                                         
  type: 'image/png',                                    
  name: 'Picture1.png',                                 
  meta: {},                                             
  ext: 'png',                                           
  extension: 'png',                                     
  extensionWithDot: '.png',                             
  mime: 'image/png',                                    
  'mime-type': 'image/png',                             
  _id: 'vu6rreiCgvBChc5Gp',                             
  userId: 'f4v3Bo4qTsdYbwFAR',                          
  path: 'C:/Test\\vu6rreiCgvBChc5Gp.png',               
  versions:                                             
   { original:                                          
      { path: 'C:/Test\\vu6rreiCgvBChc5Gp.png',         
        size: 829970,                                   
        type: 'image/png',                              
        extension: 'png' },                             
     thumbnail:                                         
      { path: 'C:/Test/thumbnail-vu6rreiCgvBChc5Gp.png',
        size: 18945,                                    
        type: 'image/png',                              
        extension: 'png',                               
        meta: [Object] } },                             
...
```

Does this make sense? It started working correctly for me after I made this fix, I think this is the expected behaviour. Happy to make a PR to the docs if you give me the green light.

**Comment by mksh-su** [comment]
Hello @leoncvlt ,

Please, make sure that your code is written correctly. You may use our [demo app](https://github.com/VeliovGroup/Meteor-Files-Demos/tree/master/demo) as a reference. A piece of code similar to yours can be found [here](https://github.com/VeliovGroup/Meteor-Files-Demos/blob/master/demo/imports/server/image-processing.js).
Anyway, don't hesitate to let us know if the code still doesn't work after you've double-checked it.

//cc @dr-dimitru

**Comment by dr-dimitru** [comment]
Hello @leoncvlt ,

I've read your message 3 times :)
Now I've get it. LGTM!

Here is 🚦(green) light!

 - Docs source is here - https://github.com/VeliovGroup/Meteor-Files/blob/master/docs/google-cloud-storage-integration.md
 - Please follow our [PR rules](https://github.com/VeliovGroup/Meteor-Files/blob/master/.github/PULL_REQUEST_TEMPLATE)
 - If possible update docs using ES6 syntax (*can be done via separate PR*)

**Comment by dr-dimitru** [comment]
Initially this tutorial is made by @sethmurphy18 , with following updates by @salmanhasni , @jeremywrnr , and @amos-whitewolf 

Guys, you're welcome to review this suggestion, and following PR.

**Comment by dr-dimitru** [comment]
@leoncvlt any news on your end, shall we wait for PR?

**Comment by salmanhasni** [comment]
https://github.com/VeliovGroup/Meteor-Files/pull/698

**Comment by dr-dimitru** [comment]
Fixed by #699, 
Feel free to reopen it in case if the issue is still persists on your end.
